example/index.ts

```diff
@@ -46,8 +46,8 @@ handleRequest('/reset-password', (req, res) => {
 });
 
 handleRequest('/transaction', (req, res) => {
-  const { subject, body } = getTransactionTemplate({ ...req.query, ...req.body })
-  res.send(subject.concat(body));
+  const { body } = getTransactionTemplate({ ...req.query, ...req.body })
+  res.send(body);
```

Comment:
Let revert this, and open another PR to change all examples

I suggest injecting `subject` prop into `<TemplateBase />`

Reply:
ok, will revert this code